packages/manager/modules/octavia-load-balancer/src/onboarding/controller.js

```diff
@@ -13,8 +27,24 @@ export default class OctaviaLoadBalancerOnboardingCtrl {
     }));
     this.illustration = illustration;
     this.cta = CTA[ovhSubsidiary] || CTA.DEFAULT;
+    this.privateNetworkHelpUrl =
+      PRIVATE_NETWORK_HELP[ovhSubsidiary] || PRIVATE_NETWORK_HELP.DEFAULT;
     this.atInternet = atInternet;
     this.ctaTrackName = `${TRACKING_CHAPTER_1}::${TRACKING_NAME}::add`;
+    this.$stateParams = $stateParams;
+    this.$http = $http;
+    this.$window = $window;
+    this.coreURLBuilder = coreURLBuilder;
+    this.missingNetworkModal = false;
+    this.hasPrivateNetwork = false;
+  }
+
+  $onInit() {
+    this.$http
```

Comment:
you can move this in routing resolve so it will be resolved before the component load and no need to inject $stateParams inside the controller doing so

Reply:
Done!